.github/workflows/test-phpunit.yml

```diff
@@ -165,33 +169,79 @@ jobs:
         uses: actions/github-script@v6
         id: phpunit-coverage-option
         with:
-          script: 'return "${{ matrix.php-versions }}" == "8.0" ? "" : "--no-coverage"'
+          script: |
+            const { COVERAGE_NAME } = process.env
+
+            return "${{ matrix.php-versions }}" == "8.0" ? `--coverage-php build/cov/coverage-${COVERAGE_NAME}.cov` : "--no-coverage"
```

Comment:
Should this be the variable instead of "8.0"?

Reply:
Yeah. The trial and error was intense so this went under my radar.